cmd/analyzer/main.go

```diff
@@ -103,6 +104,7 @@ func main() {
 		os.Exit(1)
 	}
 
+	engine.CONTEXT_LINES = *contextLines
```

Comment:
I think rather than changing a global variable, we'll want to plumb the value through the structs, similarly to `incidentLimit`. We may also need to plumb this through the providers as they can implement custom codesnippers as well.

Reply:
@fabianvf done 👍🏽